packages/anthropic/src/anthropic-messages-language-model.ts

```diff
@@ -1464,6 +1527,30 @@ export class AnthropicMessagesLanguageModel implements LanguageModelV3 {
                       cacheCreationInputTokens,
                       stopSequence,
                       container,
+                      contextManagement: contextManagement
+                        ? {
+                          appliedEdits: contextManagement.applied_edits.map(
+                            edit => {
+                              if (edit.type === 'clear_tool_uses_20250919') {
```

Comment:
switch would be better here so future additions from anthropic do not automatically get handled as clear_thinking